.gitignore

```diff
@@ -4,3 +4,5 @@
 .Ruserdata
 ggeasy.Rproj
 inst/doc
```

Comment:
I'm not sure how long this has been there (prior to this PR it seems), but FYI once I build a `pkgdown` site (soon) this will get in the way.

Reply:
Doesn't pkgdown usually build in docs folder in main dir?

Reply:
i think when we get to that bridge we can pass it and remove it from the gitignore.